### PR TITLE
docs/core: document core section

### PR DESCRIPTION
**Pull request by l-zeuch**
- docs/main: adjust weight for future chapters
- docs/main: fix image URLs in markdown documents
- docs/main: recommend reading the entire docs
- docs/core: initial work on core documentation
- docs/core: document control panel access
- docs/core: document control panel logs
- doc/core: document command settings page
- doc/core: squash typos in access settings page

**Terms**
- [x] I have read and understood this project's [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

File: content/getting-started.md
[the remark this answers is not in the document]
Comment (l-zeuch) [reply content/getting-started.md:1]: hah nice catch :sunglasses:

File: content/core/command-settings.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+when you forgot it, as simply sending `@YAGPDB.xyz prefix` will help you remember it.
+
+{{% notice style="tip" title="Flags and Switches" %}}
+
+Flags and switches are ***not*** affected by your prefix setting.
+
+For example, if your prefix is `?`, a command usage with flags and/or switches would look like the following:
+
+```
+?wyr -raw
```
Comment (savage4618) [inline content/core/command-settings.md:38]: consider expanding the "wouldyourather" command instead of the alias

**Review by savage4618 — COMMENTED** [review]
I wrote some notes on things I thought needed noting. Now I realize I probably should have done things differently after hitting the end. hindsight 20/20

File: content/core/command-settings.md
[the remark this answers is not in the document]
Comment (l-zeuch) [reply content/core/command-settings.md:1]: If we're taking a closer look in the file we see the following:
https://github.com/botlabs-gg/yagpdb-docs-v2/blob/5d153dcaec7547b9076de32490d30ecbb9e39211/content/core/command-settings.md?plain=1#L5-L9

This is for Hugo's [content summary](https://gohugo.io/content-management/summaries/#manual-summary-splitting) feature. The respective chapter's page is built with our theme's `children` shortcode, which then displays the title and said summary.

**Comment by l-zeuch** [comment]
Note to self: I should probably reference the labels established under the image at the start of the page when referring to the individual settings / sections further down.

**Pull request merged**